Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.g.props.template

```diff
@@ -119,11 +119,11 @@
   
   <ItemGroup>
 <!--SOURCE_INCLUDE_TEMPLATE_START-->
-    <Compile Include="##RELATIVE_SOURCE_PATH##" Link="##PROJECT_LINK_PATH##" />
+    <Compile Include="##RELATIVE_SOURCE_PATH_TOKEN##" Link="##PROJECT_LINK_PATH_TOKEN##" />
 <!--SOURCE_INCLUDE_TEMPLATE_END-->
 
-    <Content Include="$(ProjectGuid).csmap">
+    <!--<Content Include="$(ProjectGuid).csmap">
```

Comment:
Remove this commented code? Or is it still used anyway?

Reply:
Yep, will do that for now.